anda/langs/rust/atac/atac.spec

```
@@ -0,0 +1,47 @@
+# Generated by rust2rpm 27
+%bcond check 1
+
+%global crate atac
+
```

This uses the Oniguruma crate and that has a fun issue with GCC15's default C and C++ standards.
```suggestion

%if 0%{?fedora} >= 42
%global build_cflags %{__build_flags_lang_c} %{?_distro_extra_cflags} -std=c18 -std=gnu18
%endif
```

---

Why are you setting 2 c standards here? :thinking:

---

Because it's GCC and sometimes needs them. The first one is for the general C standard, `gnu` is for GCC extensions if they are called.

---

Hmm I'm pretty sure gnu18 is a superset of c18.

---

It's kinda long but 
> By default, GCC provides some extensions to the C language that, on rare occasions conflict with the C standard. See [Extensions to the C Language Family](https://gcc.gnu.org/onlinedocs/gcc/C-Extensions.html). Some features that are part of the C99 standard are accepted as extensions in C90 mode, and some features that are part of the C11 standard are accepted as extensions in C90 and C99 modes. Use of the -std options listed above disables these extensions where they conflict with the C standard version selected. You may also select an extended version of the C language explicitly with -std=gnu90 (for C90 with GNU extensions), `-std=gnu99` (for C99 with GNU extensions), `-std=gnu11` (for C11 with GNU extensions), `-std=gnu17` (for C17 with GNU extensions) or `-std=gnu23` (for C23 with GNU extensions).
https://gcc.gnu.org/onlinedocs/gcc/Standards.html

---

All right will look into this if this pops up.

---

It already did, that's why I noticed this used the Oniguruma Crate. https://github.com/terrapkg/packages/actions/runs/14140579658/job/39621198232#step:7:3010